ctyparser/bigcty.py

```diff
@@ -172,7 +173,9 @@ def update(self) -> bool:
 
             with tempfile.TemporaryDirectory() as temp:
                 path = pathlib.PurePath(temp)
-                dl_url = f'http://www.country-files.com/bigcty/download/{update_date[:4]}/bigcty-{update_date}.zip'  # TODO: Issue #10
+                page = session.get(update_url)
+                tree = html.fromstring(page.content)
+                dl_url = tree.xpath("//a[contains(@href,'zip')]/@href")[0]
                 rq = session.get(dl_url)
                 if rq.status_code == 404:
```

Comment:
is this codepath still necessary? I think it exists because there was a change in the feed https://github.com/miaowware/ctyparser/commit/bafa05deb30886a7e02fed43424266c0e3310f4b

Reply:
I think that fallback is needed, but very differently, and only if the download URL extraction fails (see other comment)